src/main/java/org/isf/stat/manager/JasperReportsManager.java

```diff
@@ -977,22 +993,58 @@ private JasperReportResultDto generateJasperReport(String jasperFilename, String
 		return new JasperReportResultDto(jasperPrint, jasperFilename, filename);
 	}
 
-    private String compileJasperFilename(String jasperFileName) {
-        StringBuilder sbFilename = new StringBuilder();
-        sbFilename.append("rpt");
-        sbFilename.append(File.separator);
-        sbFilename.append(jasperFileName);
-        sbFilename.append(".jasper");
-        return  sbFilename.toString();
-    }
-    
-    public String compileDefaultFilename(String defaultFileName) {
-    	StringBuilder sbFilename = new StringBuilder();
+	private String compileJasperFilename(String folderName, String jasperFileName) {
+		StringBuilder sbFilename = new StringBuilder();
+		sbFilename.append(folderName);
+		sbFilename.append(File.separator);
+		sbFilename.append(jasperFileName);
+		sbFilename.append(".jasper");
+		return sbFilename.toString();
+	}
+
+	private String compileFilename(String folderName, String jasperFileName, List<String> params, String ext) {
+		StringBuilder sbFilename = new StringBuilder();
+		sbFilename.append(folderName);
+		sbFilename.append(File.separator);
+		sbFilename.append(jasperFileName);
+		if(params != null) 
+		{
+			params.forEach(p -> {
```

Comment:
I had to hide white space differences to better see the changes in the file.   It looks like there was a fair amount of reformatting done.    In addition like things like `if(` should be `if (` to match project standards.   And braces are not on their own line, so line 1010 should be `if (parms != null) {`

Reply:
Thanks David, I reformatted the code according to the OH style